jib-core/src/test/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslatorTest.java

```diff
@@ -136,6 +141,26 @@ public void testPortListToMap() {
     Assert.assertEquals(expected, ImageToJsonTranslator.portListToMap(input));
   }
 
+  @Test
+  public void testVolumeListToMap() {
+    ImmutableList<AbsoluteUnixPath> input =
+        ImmutableList.of(
+            AbsoluteUnixPath.get("/var/job-result-data"),
+            AbsoluteUnixPath.get("/var/log/my-app-logs"));
+    ImmutableSortedMap<String, Map<?, ?>> expected =
+        ImmutableSortedMap.of(
+            "/var/job-result-data", ImmutableMap.of(), "/var/log/my-app-logs", ImmutableMap.of());
+    Assert.assertEquals(expected, ImageToJsonTranslator.volumesListToMap(input));
+  }
+
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
In our codebase, we try to avoid using `expected` and manually catch the exception in the test method body so that in most cases, we can assert the message and/or cause of the exception is as expected and that the exception was thrown from the expected location.